applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -150,81 +144,9 @@ static int do_fota_start(int op, const char *file_uri, int sec_tag,
 			 uint8_t pdn_id, enum dfu_target_image_type type)
 {
 	int ret;
-	struct http_parser_url parser = {
```

Comment:
Possibly related to above one: 
When the fota download is paused with cancel, the fota download dies with #XFOTA: 1,1,2 (ENOENT) + <err> fota_download: Download client error event.

You can reproduce this by running SLM application with following input:
at+cfun=1
at#xfota=1,"http://files.testncs.com/test/slm/9151_477c5b8d_app_update.bin"

at#xfota=0 (To pause, when the download is ongoing)
at#xfota=1,"http://files.testncs.com/test/slm/9151_477c5b8d_app_update.bin" (to continue)

Reply:
This was caused by adding a error event on calling `download_client_stop()`. I've removed that event now, as the stopped event should be enough.